### ISSUE TITLE
400.BadRequest when attempting to add UHDBits

### ISSUE THREAD

**Issue by edgeworn** [Prowlarr/Prowlarr#970]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When attempting to add UHDBits as an indexer, it gives the error "Unable to connect to indexer. We are being redirected to the login page. Most likely your session expired or was killed. Try testing the indexer in the settings." In the logs, it appears that the response is `400.BadRequest`

### Expected Behavior

I expect adding UHDBits as an indexer to work

### Steps To Reproduce

1. enable 2fa in UHDBits
2. add UHDBits as an indexer in prowlarr
3. enter username, password, and 2fa code
4. press Test or Save
5. see error: "Unable to connect to indexer. We are being redirected to the login page. Most likely your session expired or was killed. Try testing the indexer in the settings."

### Environment

```markdown
- OS: Ubuntu 20.04.4 LTS
- Prowlarr: 0.2.0.1678
- Docker Install: no
- Using Reverse Proxy: no
- Browser: Chrome 100
```

### What branch are you running?

Master

### Trace Logs?

https://0bin.net/paste/5wPii-aW#h4iTZHmwlBUk64WX2t4D3kEwPU+r5U5xXyKdJHdZagR

[AB#3053](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/3053)

**Comment by bakerboy448** [comment]
Please use one of the support channels: [Reddit](https://www.reddit.com/r/prowlarr/) or  [Discord](https://prowlarr.com/discord) for support/questions.

- there is no `master` branch of Prowlarr, not sure why you selected it
- the error is "<p>Wrong 2FA Pin!</p>";  not a bug

**Comment by edgeworn** [comment]
Sorry, you're right. This is an issue with UHDBits that I didn't notice until now. In case anyone else runs into this in the future: 2FA on UHDBits only works if you enter the 2FA code the instant that it changes to the new code. If you enter it as much as a few seconds later, it won't accept it. Tested on multiple devices. If you enter the 2FA code the instant it changes, it is possible to add it as an indexer.

And I selected `master` because the bug report form says that it is a required field with the only three options being `Master`, `Develop`, `Nightly`, and `Other (This issue will be closed)`
